modules/common.nix

```
@@ -588,6 +588,7 @@ in {
 
     passthru.inline = inline;
     passthru.catalog = builtins.toJSON newCatalog;
+    passthru.installedStorePaths = sortedStorePaths;
```

We could just call this `storePaths`, I wasn't sure if that could cause confusion with `element.storePaths`

---

Yes, i think it's better not add confusion by overloading terms
Perhaps at least add a comment what these are:

```suggestion
    # packages installed by store path,
    # for which no further information can be derived
    passthru.installedStorePaths = sortedStorePaths;
```